fat/fat2/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package fat2 provides data types corresponding to valid FAT-2 entries for
+// PegNet Transactions and Conversions as well as methods for validating the structure and
+// content of the factom entry.
```

Comment:
Is there a line length policy you are following for these? Not sure I am aware of one for go, but will follow whatever convention you have set for the project

Reply:
There isn't one for go but I try to keep line lengths at 79 chars max. I'll add this to the code policy doc.

Reply:
The one exception to this is that I try to avoid breaking up error strings so that they are easier to search for in the code base. But in general I try to find other formatting ways to keep the line length down in such cases.